cmd/storagenode-updater/main.go

```diff
@@ -112,10 +119,15 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 	}()
 
 	loopFunc := func(ctx context.Context) (err error) {
-		if err := update(ctx, ident.ID); err != nil {
+		if err := update(ctx, runCfg.BinaryLocation, runCfg.ServiceName, renameStoragenode); err != nil {
 			// don't finish loop in case of error just wait for another execution
 			log.Println(err)
 		}
+
+		if err := update(ctx, os.Args[0], updaterServiceName, renameUpdater); err != nil {
+			// don't finish loop in case of error just wait for another execution
+			log.Printf("%+v", err)
```

Comment:
Why printing here differs from the way we prit whe updating the storage node binary?

Reply:
no idea, I will change it to log.Printf(err)